SeaPublicWebsite/Views/EnergyEfficiency/Recommendations/AlternativeRecommendations.cshtml

```diff
@@ -0,0 +1,128 @@
+    @await Html.GovUkBackLink(new BackLinkViewModel
+    {
+        Text = "Back",
+        Href = Model.BackLink
+    })
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        
+        <h1 class="govuk-heading-l">
+            Other ways to save energy
+        </h1>
+        
+        <p class="govuk-body">
+            If you are looking for some easy to implements reccomendations or some alternatives incase the ones we 
+            presented were not suitable, here are some other actions  you could consider.
+        </p>
+        <p class="govuk-body">
+            Or you can proceed straight to your
+            <a class="govuk-link" href="@Url.Action(nameof(EnergyEfficiencyController.ActionPlan_Get), "EnergyEfficiency", new {reference = Model.Reference})">action plan</a> 
+            for a summary of the recommendations 
+            you would like to act on and advice on your next steps.
+        </p>
+        
+        @if (Model.PropertyData.PropertyRecommendations.Exists(r => 
+            r.Key is RecommendationKey.InsulateSolidWalls or RecommendationKey.InsulateCavityWalls))
+        {
+            <h2 class="govuk-heading-m">
+                Reflective radiator panels
+            </h2>
+            
+            <p class="govuk-body">
+                Because you have uninsulated walls, you could save energy by fitting reflective radiator panels 
+                behind radiators on external walls. More information is available at the 
+                <a class="govuk-link" href="https://energysavingtrust.org.uk/" target="_blank">Energy Saving Trust website.</a>
+            </p>
+        }
+        
+        @if (Model.PropertyData.HeatingType is not HeatingType.HeatPump)
+        {
+            <h2 class="govuk-heading-m">
+                Heat your home with a heat pump
+            </h2>
+            
+            <p class="govuk-body">
+                It may be possible to replace your current heating system with a heat pump, a low carbon heating 
+                system. More information is available at 
+                <a class="govuk-link" href="https://www.heat-pump-check.service.gov.uk/" target="_blank">Check if a heat pump could be suitable for your home.</a>
+            </p>
+        }
+        
+        @if (Model.PropertyData.FloorInsulated is FloorInsulated.No)
+        {
+            <h2 class="govuk-heading-m">
+                Draught-proof your floors
+            </h2>
+            
+            <p class="govuk-body">
+                Insulating your floor can be disruptive, but draught-proofing doesn’t need to 
+                be and can still help reduce your energy bills. More information available at the
+                <a class="govuk-link" href="https://energysavingtrust.org.uk/" target="_blank">Energy Saving Trust website.</a>
+            </p>
+        }
+        
+        @if (Model.PropertyData.GlazingType is GlazingType.SingleGlazed or GlazingType.Both)
```

Comment:
Similar to before, this is business logic and shouldn't really be in a cshtml file